nemo/collections/asr/parts/utils/transcribe_utils.py

```diff
@@ -227,10 +226,8 @@ def prepare_audio_data(cfg: DictConfig) -> Tuple[List[str], bool]:
                 else:
                     has_two_fields.append(False)
                 audio_key = cfg.get('audio_key', 'audio_filepath')
-                audio_file = Path(item[audio_key])
-                if not audio_file.is_file() and not audio_file.is_absolute():
-                    audio_file = manifest_dir / audio_file
-                filepaths.append(str(audio_file.absolute()))
+                audio_file = get_full_path(audio_file=item[audio_key], manifest_file=cfg.dataset_manifest)
```

Comment:
so we need both audio_dir and manifest to locate the path of the audio file just like in your example file? 

`{"audio_filepath": ["61-70968-3-4-5_ch_0.wav", "61-70968-3-4-5_ch_1.wav", "61-70968-3-4-5_ch_2.wav"], "text": "he was like unto my father in a way and yet was not my father", "offset": 0, "duration": 4.32}`

Reply:
oh I see! Thanks for clarification!